### PR TITLE
Translated glamor.md

### PR DESCRIPTION
**Pull request by akramkazmi71**

### CODE REVIEW

File: docs/docs/glamor.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 title: Glamor
 ---
 
-CSS-in-JS library [Glamor](https://github.com/threepointone/glamor) is not actively maintained. the maintainer recommends using [Emotion](/docs/emotion/).
+CSS-in-JS लाइब्रेरी में [Glamor](https://github.com/threepointone/glamor) सक्रिय रूप से बनाई नहीं रखी गई है। मेंटेनर [Emotion](/docs/emotion/) का उपयोग करने की सलाह देता है।
```
Comment (arshadkazmi42) [inline docs/docs/glamor.md]: `बनाई नहीं रखी गई` => `मेन्टेन नहीं की जाती`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Added one small review

File: docs/docs/glamor.md (hunk 1 above)
Comment (akramkazmi71) [reply docs/docs/glamor.md]: Done with the changes.

**Review by arshadkazmi42 — APPROVED** [review]
👍

**Pull request merged**